procrastinate/job_context.py

```diff
@@ -30,26 +32,45 @@ class JobContext:
     app: Optional[app_module.App] = None
     worker_name: Optional[str] = None
     worker_queues: Optional[Iterable[str]] = None
+    worker_id: int = 0
```

Comment:
I am wondering if the `worker_id` shouldn't be set to `-1` in the base context.

Reply:
I'd rather have it None that -1. People would not expect -1.